Sources/PointFree/SubscribeConfirmation.swift

```diff
@@ -33,7 +33,7 @@ public let subscribeConfirmation: Middleware<
             coupon,
             currentUser,
             Current.stripe.js,
-            Current.envVars.stripe.publishableKey.rawValue
+            Current.envVars.stripe.publishableKey
```

Comment:
I upgraded the middleware to take a `Stripe.Client.PublishableKey` instead of just the raw string value.